src/components/ItemDetail/ItemDetail.js

```diff
@@ -70,6 +70,8 @@ const ItemDetail = props => {
     };
   };
 
+  const primoLink = `https://search.library.northwestern.edu/primo-explore/search?field=any&query=any,contains,${catalogKey}&query=&institution=01NWU&vid=NULVNEW&search_scope=NWU`;
```

Comment:
@kdid Think we'd use this anywhere else?  If so, maybe move to the `globalVars.js` file?   If not, all good.

Reply:
I don't see where we'd use it anywhere else at the moment, but I suppose it's possible.

What were you thinking in terms of storing it in `global-vars.js` file? The first part of the url before the `catalog_key` and the part of the url after it would be stored separately?

Reply:
Sorry, didn't see that variable in there.  If not being used elsewhere, use as you have it I'd say.  If a second one pops up, can abstract it out to a helper function or something.